includes/blocks/course-theme/class-next-lesson.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * File containing the Next_Lesson class.
+ *
+ * @package sensei
+ * @since
+ */
+
+namespace Sensei\Blocks\Course_Theme;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use \Sensei_Blocks;
+
+/**
+ * Class Next_Lesson is responsible for rendering the 'Next Lesson >' blocks.
+ */
+class Next_Lesson {
+
+	/**
+	 * The right chevron icon.
+	 *
+	 * @var string
+	 */
+	public static $icon = '<svg viewBox="0 0 8 13" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Note that we also have a solution in the course outline block with svg `<use>` tags (eg. `<svg><use xlink:href="#sensei-chevron-up"></use></svg>`), linking to a [library we render](https://github.com/Automattic/sensei/blob/dd4c93373c569ddedb1af049645fe5bf17dae7c4/includes/blocks/class-sensei-course-outline-course-block.php#L23-L35) at the end of the block.

I was just thinking to expand that to a helper where render functions can just use the icon and at the end we output the icons needed on the current page.

Reply:
Agree. I implemented a simple helper `Sensei_Utils::icon( string $name = '' )`. Currently it only retrieves the content of the icon file at `assets/images/<name>.svg` or empty string if not found. But we can expand it further and introduce optimizations like @yscik mentioned above.

The implementation is here #4432 . Since this branch is already merged I figured I'd address the comments here in that PR. Sorry for the mess guys.